test_file.py

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env python3
+
+print("Hello world!")
```

Comment:
Any reason why you don't have a comment indicating what your code is doing?

Reply:
I guess I wasn't sure how much commenting to include for this. What would be your preference?